pkg/registry/ggcr/client.go

```diff
@@ -32,8 +51,12 @@ type client struct {
 	writeRemoteIndex indexWriter
 }
 
+var _ RegistryClient = &client{}
+
+// TODO: *client implements Digest and Copy too
```

Comment:
Shall we address this TODO in this PR as well?

Reply:
It was really a note to self and I've deleted it now.

Reply:
That interface looked like dead code and I inadvertently deleted it. https://github.com/pivotal/image-relocation/pull/39 will re-instate it (and ensure it isn't dead so I don't make the same mistake again). Apologies!